CouchBaseStorage/CouchBaseStorageProvider.cs

```diff
@@ -61,12 +64,17 @@ public class CouchBaseDataManager : IJSONStateDataManager
         /// </summary>
         protected IBucket bucket;
 
+        /// <summary>
+        /// 
+        /// </summary>
+        private Dictionary<string, TimeSpan> DocumentExpiries { get; }
+
         /// <summary>
         /// Constructor
         /// </summary>
         /// <param name="bucketName">Name of the bucket that this manager should operate on.</param>
         /// <param name="clientConfig">Configuration object for the database client</param>
-        public CouchBaseDataManager(string bucketName, Couchbase.Configuration.Client.ClientConfiguration clientConfig)
+        public CouchBaseDataManager(string bucketName, Couchbase.Configuration.Client.ClientConfiguration clientConfig, Dictionary<string, TimeSpan> documentExpiries)

```

Comment:
If we added a new constructor overload that took the document expiries it would mean we didn't have to change `CouchBaseMembershipProvider` or `CouchBaseDataManagerTests`.